server/app.ts

```diff
@@ -45,8 +45,8 @@ export default function createApp(services: Services, applicationInfo: Applicati
   app.use(setUpAuthentication())
   app.use(authorisationMiddleware())
   app.use(setUpCsrf())
```

Comment:
moved populating current user details below populating client token as we need client token to fetch staff details in order to populate staff details like id, code, etc.

Reply:
but will the username exist on the request.user object? to send through when requesting the client token?

Reply:
**answer**: Yes, it is set [here](https://github.com/ministryofjustice/hmpps-assess-for-early-release-ui/blob/31d0be8f9ee6cb0933600827cfdf5123b6a29d22/server/middleware/setUpAuthentication.ts#L55) before that.